mtags/src/main/scala-3/scala/meta/internal/pc/completions/Completions.scala

```diff
@@ -578,6 +578,7 @@ class Completions(
           indexedContext.lookupSym(sym) match
             case IndexedContext.Result.InScope =>
               visit(CompletionValue.scope(sym.decodedName, sym))
+            case _ if sym.maybeOwner.is(Flags.CaseClass) => true
```

Comment:
It doesn't make sense for any class not just `case class` but never happens for anything else, I am understanding it right?

Reply:
Turns out it happens more ofter. We are indexing also members of case class/class/trait etc. but we don't filter them out for completions, and they can't be brought to scope via import. I changed `isAccessible`, which should solve this issue